Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -172,12 +172,14 @@ def change_key_names(data):
 def container_name(data):
     """ Retrieve container name from information about dataset.
 
-    As AMI GetPhysicsParamsForDataset works with containers,
-    we construct the container name from each dataset name,
-    removing the _tid{...} part
+    The container name is extracted from dataset name by removing
+    the '_tid...' part.
 
-    :param dataset: dataset information
-    :return: dataset name without _tid => container name
+    :param data: dataset information
```

Comment:
Parameter name has actually changed an another commit (12d5ee9).
It is fixed here (4f88409), among other improvements of the docstrings, but you still might want to amend previous commit or reword this one, mentioning the parameter name fixing. Not a big deal, maybe, but it catches the eye since you have a pretty detailed list of changes and this one is not mentioned.

Reply:
I missed this, thanks for pointing out. Moved the change to eddad82 (former 12d5ee9).